lib/liquid/strainer.rb

```diff
@@ -22,12 +24,14 @@ def initialize(context)
     end
 
     class << self
+      attr_accessor :strainer_class_cache, :global_strainer
```

Comment:
This is necessary because subclasses otherwise can't access those variables anymore?

Reply:
This was all part of switching from class variables to class instance. This seemed to be the pattern to allow it to still access them. I have to admit I'm not 100% on this implementation I was just tweaking on what allowed the test suite to pass and this format seems to work.